mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -213,6 +214,7 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "withdraw" => handle_mmrpc(ctx, request, withdraw).await,
         "ibc_chains" => handle_mmrpc(ctx, request, ibc_chains).await,
         "ibc_transfer_channels" => handle_mmrpc(ctx, request, ibc_transfer_channels).await,
+        "peer_connection_healthcheck" => handle_mmrpc(ctx, request, peer_connection_healthcheck_rpc).await,
```

Comment:
Could you plz clarify how this rpc is used by GUI? (Maybe it should be a draft doc note for QA team.)
Is it for checking that another party is online during the swap?

Reply:
This is a general-purpose RPC method to check if some peer is currently active in the kdf network or not. Currently it is utilized from our proxy application. You can see how it works from the README at https://github.com/KomodoPlatform/komodo-defi-proxy/tree/1294ab27b868c9d3e3ebabf70a8e469652b78513.

It's quite straightforward RPC which takes one parameter and returns a boolean flag. We should probably include that in the KDF dev page along with the other RPCs. cc @KomodoPlatform/qa